pkg/internal/checkup/executor/executor.go

```diff
@@ -89,7 +89,7 @@ func (e Executor) Execute(ctx context.Context, vmiUnderTestName, trafficGenVMINa
 		return status.Results{}, fmt.Errorf("failed to Start to Trex Service on VMI \"%s/%s\": %w", e.namespace, trafficGenVMIName, err)
 	}
 
-	trexClient := trex.NewClient(e.vmiSerialClient, e.namespace, e.verbosePrintsEnabled)
+	trexClient := trex.NewClient(e.vmiSerialClient, e.namespace, e.trafficGeneratorPacketsPerSecond, e.testDuration, e.verbosePrintsEnabled)
```

Comment:
Could you please add a few words regarding the function of the above parameters in the commit message?

Reply:
Not much to say really. I wouldn't have moved them if the lint wouldn't have complained about it.